pom.xml

```diff
@@ -316,6 +316,32 @@ THE SOFTWARE.
               <overWriteSnapshots>true</overWriteSnapshots>
             </configuration>
           </execution>
+          <execution>
+            <id>copy-fips-libs</id>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.bouncycastle</groupId>
+                  <artifactId>bc-fips</artifactId>
+                  <version>1.0.2.4</version>
```

Comment:
you may want a TODO here, as this is not the latest version of `bc-fips` but for a valid reason.